docs/src/main/asciidoc/config.adoc

```diff
@@ -488,6 +488,30 @@ quarkus.datasource.url=jdbc:mysql://${application.server}:3306/mydatabase?useSSL
 
 It does result in one more line in this example but the value of `application.server` can be reused in other properties, diminishing the possibility of typos and providing more flexibility in properties definitions.
 
+=== Combining placeholders and environment variables
+
+Quarkus also supports the combination of both placeholders and environment variables.
+
+Let's assume you have following property defined in `application.properties`
+[source,properties]
+----
+remote.host=quarkus.io
+----
+
+You can combine environment variables and placeholders by having a property defined as follow
+[source,properties]
+----
+application.host=${HOST:remote.host}
+----
+
+This will expand the HOST environment variable and use the value of the property `remote.host` as the default value if `HOST` is not set.
```

Comment:
Actually, this didn't work for me. To get the effect that is described, I had to do:

```
application.host=${HOST:${remote.host}}
``` 

Which also means that something like this would even work:

```
application.host=${HOST:${remote.host:localhost}}
```

Would you like to give it a shot and update if you see the same effect?